vunit/sim_if/modelsim.py

```diff
@@ -410,9 +410,9 @@ def encode_generic_value(value):
     """
     s_value = str(value)
     if " " in s_value:
-        return '{"%s"}' % s_value
+        return f'{{"{s_value!s}"}}'
     if "," in s_value:
-        return '"%s"' % s_value
+        return f'"{value!s}"'
```

Comment:
I don't think `value` or `s_value` matters here but line 413 and 415 should be consistent.

Reply:
Fixed!